modular_skyrat/master_files/code/modules/clothing/under/jobs/command.dm

```diff
@@ -145,6 +145,11 @@
 *	MISC
 */
 
+/obj/item/clothing/under/rank/captain/skyrat/baseball
```

Comment:
Why is it a captain subtype exactly?

Reply:
All generic command gear is 🤷 God bless SS13

I'll probably toss this upstream anyways so it'll most likely end up moving to a greyscale costume. Gonna leave this PR up for da reviews tho 😏